src/components/ContentHighlights/ContentHighlightCardItem.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Card } from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+const ContentHighlightCardItem = ({ title, type }) => (
+  <Card
+    isClickable
+    className="h-100"
+  >
+    <Card.ImageCap
+      className="h-100"
+      src="https://source.unsplash.com/360x200/?nature,flower"
+      srcAlt="Card Image"
+      logoSrc="https://via.placeholder.com/150"
+      logoAlt="placeholder"
+    />
+
+    <Card.Header
+      className="h-100"
```

Comment:
These values already existed on the card I was templating from learner portal, removed all instances of them from the card since we are going to be templating this component out for future use.